### DISCUSSION TITLE
Perfekt mulighet for reklame for Entur/Tavla

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#562] [Ideas]
På Jernbanetorget har det i minst et halvt år ikke vært mulig å se når banen østover går. Det hang en liten skjerm der om det, men den har altså forsvunnet for lenge siden. Og selv vestgående er vanskelig å se, fordi skjermen er så liten og vanskelig plassert. Samtidig er det en enorm skjerm der, som er rett fram der man vil se etter ruteinformasjonen. Men denne brukes av uforståelige grunner kun til et kart (som kunne vært papir), og reklame. 

![image](https://pbs.twimg.com/media/FETVChZXoAEBjGS?format=jpg&name=large)

Jeg har klaget til Ruter ganske mye om dette på Twitter, og fått mye støtte, og jeg har uttalt meg til media i denne saken: https://www.ao.no/skjermen-har-vart-borte-i-et-halvt-ar-det-er-ille/s/5-128-218033

Entur kjøper reklame diverse steder har jeg sett. Så tanken min er at det hadde vært helt perfekt om Entur kjøpte reklametid på denne skjermen og kun viste en oversikt over når banen østover går – samt en logo (+ tagline) selvsagt. Og gjerne i form av en Tavla-tavle.

Journalisten som skrev saken over har sagt at han er interessert i å lage en oppfølger-sak, siden det ikke kom så mye ut av denne, så der finnes det et potensiale for litt medieoppmerksomhet også.

Jeg har til og med [laget en tavle for dere](https://tavla.entur.no/t/Tef98p7KzIZ8s0NBzqNo), som er klar til bruk (utenom at jeg ikke aner om 5 Storo går vestover eller østover. Det vil løse seg om dere implementerer #561. Foreløpig har jeg latt den stå på. Den vil også bli bedre om dere fikser #552.)

Hva syns dere? Jeg syns ihvertfall dette hadde både vært nyttig, og veldig gøy! Og jeg hadde elsket å gå å ta bilde av dette og poste sammen med alle klagene mine til Ruter på Twitter :P

**Comment by forteller** [comment]
Ojsann, der var denne muligheten plutselig borte. Men det er en veldig kjent bakgrunnsfarge på den nye ruteinfoen på skjermen… 🤔 Dere har vel ikke tilfeldigvis noe med dette å gjøre dere? https://twitter.com/forteller/status/1492181122041753604

**Reply by draperunner** [reply]
Det var nok ikkje oss, men fargen er fin! 🤩

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
